screens/ProfileScreen.tsx

```diff
@@ -179,7 +179,7 @@ export default function ProfileScreen({
         {
           text: "Ja",
           onPress: () => {
-            deleteUser(userToDelete);
+            removeAccount();
             Alert.alert("Ditt konto har raderats");
             setModalVisible(false);
             navigation.navigate("Signin");
```

Comment:
I discovered a bug. When you delete your account you dont get redirected to signin but you are still in profileStack an Sign in is only added as the latest screen in the stack. The navbar is still there and you can switch between the different tabs. 

![Simulator Screen Shot - iPhone SE (3rd generation) - 2023-01-25 at 11 47 48](https://user-images.githubusercontent.com/80100462/214544003-e4771663-a91d-4b07-a5f7-a982221aa852.png)